### PR TITLE
Updated variable names & added doc strings

### PR DESCRIPTION
**Pull request by emilioward**
## What? and Why?
Refactored variable names to make them easily readable and added doc strings to make classes easier to understand what their function is. 

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by emilioward** [comment]
[MWSS Transformer Documentation (3)](https://trello.com/c/HA9XKToQ/446-mwss-transformer-documentation-3)

File: tests/test_mwss_transform.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def test_weekly_increase(self):
         Increase in weekly pay (100).
 
         """
-        dflt, fn = MWSSTransformer.ops()["100"]
+        default, fn = MWSSTransformer.ops()["100"]
```
Comment (gemmai95) [inline tests/test_mwss_transform.py]: Could fn be changed to function?

File: tests/test_mwss_transform.py [hunk 2]
```diff
@@ -334,62 +334,62 @@ def test_aggregate_monthly_changes(self):
...
         self.assertEqual(125000, result)
-        for qid in ("140m", "140w4", "140w5"):
-            result = fn("140", {"140": "125000", qid: "25000"}, 0)
+        for question_id in ("140m", "140w4", "140w5"):
+            result = fn("140", {"140": "125000", question_id: "25000"}, 0)
             self.assertEqual(150000, result)
 
     def test_aggregate_fiveweekly_changes(self):
         """
         QIds 190w5 - 197w5 used for fiveweekly changes questions; all aggregated as 190.
```
Comment (gemmai95) [inline tests/test_mwss_transform.py]: Change docstring to match code. Question ID

File: tests/test_mwss_transform.py [hunk 3]
```diff
@@ -592,57 +592,57 @@ def test_currency(self):
...
-        for q, val in pay_frequencies.items():
-            qid = str(q)
-            with self.subTest(qid=qid, val=val):
-                result = MWSSTransformer.transform({qid: val})
-                self.assertIs(True, result[qid])
-                self.assertEqual(1, CSFormatter.pck_value(qid, result[qid]))
-                result = MWSSTransformer.transform({qid: ""})
-                self.assertIs(False, result[qid])
-                self.assertEqual(2, CSFormatter.pck_value(qid, result[qid]))
+        for q, result in pay_frequencies.items():
```
Comment (gemmai95) [inline tests/test_mwss_transform.py]: Could q be renamed?

File: tests/test_mwss_transform.py [hunk 4]
```diff
@@ -97,16 +97,16 @@ def test_weekly_increase(self):
         Increase in weekly pay (100).
 
         """
-        dflt, fn = MWSSTransformer.ops()["100"]
+        default, fn = MWSSTransformer.ops()["100"]
```
Comment (JamesGardiner) [inline tests/test_mwss_transform.py]: Could fn be something more descriptive? Maybe `funct`?

File: tests/test_mwss_transform.py [hunk 5]
```diff
@@ -221,81 +221,81 @@ def test_aggregate_fortnightly_increase_employees(self):
...
 
         """
-        dflt, fn = MWSSTransformer.ops()["90"]
-        for qid in ("90w", "91w", "92w", "93w", "94w", "95w", "96w", "97w"):
-            with self.subTest(qid=qid):
-                result = fn("90", {qid: ""}, True)
+        default, fn = MWSSTransformer.ops()["90"]
+        for question_id in ("90w", "91w", "92w", "93w", "94w", "95w", "96w", "97w"):
+            with self.subTest(question_id=question_id):
+                result = fn("90", {question_id: ""}, True)
```
Comment (JamesGardiner) [inline tests/test_mwss_transform.py]: Further up this was `question`: I think `question_id` is better.

File: tests/test_mwss_transform.py [hunk 6]
```diff
@@ -577,11 +577,11 @@ def test_unsigned(self):
     def test_unsigned_decimals(self):
         digits_ingested_as_bools = [100, 200]
```
Comment (JamesGardiner) [inline tests/test_mwss_transform.py]: Docstrings

File: tests/test_mwss_transform.py [hunk 7]
```diff
@@ -577,11 +577,11 @@ def test_unsigned(self):
...
-                self.assertIs(True, result[qid])
-                self.assertEqual(1, CSFormatter.pck_value(qid, result[qid]))
+            question_id = str(qNr)
+            with self.subTest(qNr=qNr, question_id=question_id):
+                result = MWSSTransformer.transform({question_id: "64.0"})
+                self.assertIs(True, result[question_id])
+                self.assertEqual(1, CSFormatter.pck_value(question_id, result[question_id]))
 
     def test_currency(self):
         result = MWSSTransformer.transform({"50": "36852"})
```
Comment (JamesGardiner) [inline tests/test_mwss_transform.py]: Docstring

File: transform/transformers/MWSSTransformer.py [hunk 8]
```diff
@@ -136,8 +144,8 @@ def main(args):
 def run():
     parser = sdx.common.cli.transformer_cli(__doc__)
     args = parser.parse_args()
-    rv = main(args)
-    sys.exit(rv)
+    result = main(args)
```
Comment (JamesGardiner) [inline transform/transformers/MWSSTransformer.py]: `return_value` is probably a more accurate variable name

File: tests/test_mwss_transform.py [hunk 9]
```diff
@@ -592,57 +592,57 @@ def test_currency(self):
...
+                self.assertEqual(2, CSFormatter.pck_value(question_id, result[question_id]))
                 result = MWSSTransformer.transform({})
-                self.assertIs(False, result[qid])
-                self.assertEqual(2, CSFormatter.pck_value(qid, result[qid]))
+                self.assertIs(False, result[question_id])
+                self.assertEqual(2, CSFormatter.pck_value(question_id, result[question_id]))
 
     def test_dates_to_onetwo(self):
         dates_ingested_as_bools = [110, 210]
         for qNr in dates_ingested_as_bools:
```
Comment (dcdarrell9) [inline tests/test_mwss_transform.py]: is it worth changing qNr also?

**Comment by emilioward** [comment]
Please do not delete this branch

**Pull request merged**